tests/std/tests/P0645R10_text_formatting_formatting/test.cpp

```diff
@@ -1520,6 +1520,23 @@ void test_gh_4319() {
     assert(format(STR("{:}"), 12345678.0) == STR("12345678"));
 }
 
+// GH-4320: <format>: crash when formatting a floating-point value with #/L and a large precision
+void test_gh_4320() {

```

Comment:
Should we also test the `wchar_t` version?

Reply:
IMO it's unnecessary. We know that the affected code path does not depend on the character type.

Reply:
But I still add it, `we know` doesn't means still same in the future.